ts/config.ts

```diff
@@ -1,3 +1,3 @@
 export const config = {
-  requestValidityPeriod: 10000 // milliseconds
+  requestValidityPeriod: 1 * 1000 // milliseconds
```

Comment:
It's too simple. I would like to add some ability to control this value in server and request methods as option. Please look at the patch file?
[Request_Validity_Period_as_option.zip](https://github.com/contentstack/epicurus-node/files/6922351/Request_Validity_Period_as_option.zip)

Reply:
Done.